src/StoreKeeper/WooCommerce/B2C/Tools/TaskRateCalculator.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Tools;
+
+use StoreKeeper\WooCommerce\B2C\Models\TaskModel;
+
+class TaskRateCalculator
+{
+    private $tasks;
+    private $minuteMetrics;
+    private $startDateTime;
+    private $endDateTime;
+
+    public function __construct(array $tasks, $now = null, $minuteMetrics = 60)
```

Comment:
```suggestion
    public function __construct(array $tasks, ?string $now = null)
```
Shouldn't `$minuteMetrics` be a const? Why is it a paramter?

Reply:
Added it as an optional parameter so that it can be easily overridden wherever taskcalculator will be used, but I changed it to const now.